dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -106,13 +139,25 @@ public static void registerZenodoDOIForWorkflow(String zenodoUrl, String docksto
                 // accessed through the "latest_draft" under "links" in the response body.
                 Object links = returnDeposit.getLinks();
                 String depositURL = (String)((LinkedHashMap)links).get("latest_draft");
-
                 String depositionIDStr = depositURL.substring(depositURL.lastIndexOf("/") + 1).trim();
                 // Get the deposit object for the new workflow version DOI
                 depositionID = Integer.parseInt(depositionIDStr);
                 returnDeposit = depositApi.getDeposit(depositionID);
 
                 depositMetadata = returnDeposit.getMetadata();
+
```

Comment:
This block looks exactly the same as the block starting line at 114. It also seems that there is other code in the greater block that could be de-duplicated, but at least this new block shouldn't be duplicated.

Reply:
Not sure which block is being mentioned...

Reply:
Currently lines 109-116 = lines 147-154.

Reply:
Created a method createAliasAndsetUpRelatedIdentifiers to encompass the common code